doc/api/v8.md

```diff
@@ -135,6 +135,31 @@ to get swapped out by the operating system.
 }
 ```
 
+## v8.heapdump.getHeapdump()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object}
+
+Returns an object containing a snapshot of the current V8 heap.
```

Comment:
Maybe there is no good documentation on this yet, but it would be great if the following two were mentioned:

- structure of the object
- how do I use this object?

Reply:
Yeah, I'm going to be asking about the stability of this object structure to see about documenting it.

Reply:
How could somebody write this to disk?

Reply:
> How could somebody write this to disk?

By using `v8.heapdump.triggerHeapdump()` :-)

Reply:
The object structure is not documented and is entirely V8 specific. However, it has also been stable for some time. My recommendation here is that we simply note that the the data is intended to be used with Chrome DevTools.